sdk/cosmosdb/cosmos/src/queryExecutionContext/defaultQueryExecutionContext.ts

```diff
@@ -133,11 +131,11 @@ export class DefaultQueryExecutionContext implements ExecutionContext {
     try {
       let p: Promise<Response<any>>;
       if (this.nextFetchFunction !== undefined) {
-        log.debug("using prefetch");
+        logger.info("Debug: using prefetch");
```

Comment:
is it possible to only log certain log messages conditionally for debug level? meaning if the log level is not debug then we don't log certain messages?

doesn't azure logger, has something like` logger.debug` that we can use here?

Reply:
It's not there.

Reply:
let me check what others are doing.

Reply:
It's automatic...any logs that are written using a log level equal to or less than the one you choose will be emitted.

Reply:
... log level equal to or less than the one you choose will be emitted. sounds like it will be chatty at any level above debug.
error, warning, verbose, info

Reply:
thanks for explanation. In that case, if I understood correctly, it seems verbose is the closest to debug.

so as the old code was debug, isn't verbose more suitable here?

```javascript
log.verbose("using prefetch");
```

I wonder if we can choose verbose here instead as it seems a more suitable replacement for debug?